cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -657,10 +709,10 @@ private void configureAuthorization(String clusterName, List<String> superUsers,
      *
      * @param authorization     Simple authorization configuration
      * @param superUsers        Super-users list who have all the rights on the cluster
-     * @param useKRaft          Use KRaft mode in the configuration
+     * @param isKRaft          Use KRaft mode in the configuration
```

Comment:
Same as above.

Reply:
This parameter was removed from the builder, because not needed anymore.